eclipse-cdt-cloud/vscode-memory-inspector
# Memory Table Editability

#### What it does


An alternative implementation of #108 that allows both editing by group (by double clicking on the group) and editing of sub- and supra-group editing by selection and context menu.

#### How to test


> As in #108:

1. Double click on a data group within the table
2. Provide a different value
3. submit with enter or blur the element
4. Changes should be applied

> In addition:

1. Select text in a data column row.
2. Use the context menu and select `Edit currently selected memory`
3. All words whose text was included in the selection will be converted into a text input.

![image](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/assets/62660806/7fcc767b-b2de-440a-86b9-c2773cee8962)

![image](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/assets/62660806/958545b9-22f7-4679-9db5-424f0a54a89d)

![image](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/assets/62660806/d2a536be-5591-4747-a804-5b1437ccd462)

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
